scripts/build.py

```diff
@@ -112,6 +115,19 @@
 HASH_BLOCK_SIZE = 2**20
 
 
+def test_compiled_js_dir_is_valid():
```

Comment:
Let's call this require_compiled_js_dir_to_be_valid() -- we only prefix stuff with test_ in the test files. 

Please add Raises docstring, too.

Reply:
Done.